lib/packages.cf

```diff
@@ -32,13 +32,22 @@ bundle common package_module_knowledge
     windows::
       "platform_default" string => "msiexec";
 @endif
+
+@if minimum_version(3.12.2)
+    termux::
+      "platform_default" string => "apt_get";
+@endif
 }
 
 body package_module apt_get
 {
     query_installed_ifelapsed => "$(def.package_module_query_installed_ifelapsed)";
     query_updates_ifelapsed => "$(def.package_module_query_updates_ifelapsed)";
     #default_options =>  {};
+@if minimum_version(3.12.2)
```

Comment:
This is required because prior to 3.12.2, interpreter was not understood by the language.

Reply:
Note, this will break on 3.13.x

Reply:
Can I do complex expressions then? Will check. :+1:

Reply:
How about 3.15.1? That's the earliest termux version and handles the situation nicely.

https://github.com/termux/termux-packages/commits/master/packages/cfengine